src/matrixHelpers/formioScript/createScripts.js

```diff
@@ -0,0 +1,15 @@
+export const createScripts = (scripts) => {
+  scripts.forEach(({ type, async, src, href, rel }) => {
```

Comment:
should we not check for script exists before appending or does it not matter to double load?

Reply:
the next if condition is for checking script does exist or not

Reply:
ah, cool am not fully up to speed on querySelectors. 

Also, we don't need to support ie11 any more. 



Browser | Usage | Notes
-- | -- | --
Safari | 39.21% (down from 39.23%) | Version 12 and laterThis browser has high standards compliance and should be easy to support
Chrome | 38.98% (down from 41.09%) | Version 91 and laterThis browser has high standards compliance and should be easy to support
Safari (in-app) | 5.75% (new to list) | unknown version, accessed within an app (e.g. Facebook iOS app)
Microsoft Edge | 5.55% (up from 5.38%) | Version 90 or laterDefault browser for all Windows 10 devices
Samsung Internet | 4.81% (down from 5.56%) | Version 11 or laterDefault browser for Samsung devices running Android 6.0 and later
Android Webview | 2.79% (new to list) | Version 92 or laterLaunched from within an app (e.g. Facebook Android app)
Firefox | 2.1% (down from 2.3%) | Version 88 and laterThis browser has high standards compliance and should be easy to support



https://www.forgov.qld.gov.au/information-and-communication-technology/communication-and-publishing/website-and-digital-publishing/website-standards-guidelines-and-templates/browser-support/browser-support-list